crates/bevy_render/src/render_resource/resource_macros.rs

```diff
@@ -0,0 +1,102 @@
+#[cfg(debug_assertions)]
+#[macro_export]
+macro_rules! render_resource_wrapper {
```

Comment:
Nice! This is much better than the previous macros.

Reply:
taking the dog for a walk always yields better code. thanks for the suggestions.